src/ensembl/src/content/app/browser/track-panel/SidebarToolstripCollection.tsx

```diff
@@ -0,0 +1,8 @@
+export enum SidebarToolstripCollection {
+    SEARCH = 'search',
+    BOOKMARKS = 'bookmarks',
+    TRACKS_MANAGER = 'tracks-manager',
+    PERSONAL_DATA = 'personal-data',
+    SHARE = 'share',
+    DOWNLOADS = 'downloads'
+  }
```

Comment:
There is already a duplicate file in this PR called `src/ensembl/src/shared/types/sidebar-toolstrip-collection.ts`